library/Ivoz/Provider/Domain/Service/Company/ValidateBillingMethod.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Ivoz\Provider\Domain\Service\Company;
+
+use Ivoz\Core\Application\Service\EntityTools;
+use Ivoz\Provider\Domain\Model\Company\CompanyDto;
+use Ivoz\Provider\Domain\Model\Company\CompanyInterface;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+
+class ValidateBillingMethod implements CompanyLifecycleEventHandlerInterface
+{
+    public const PRE_PERSIST_PRIORITY = self::PRIORITY_NORMAL;
+
+    public function __construct(
+        private EntityTools $entityTools,
+    ) {
+    }
+
+    public static function getSubscribedEvents()
```

Comment:
```suggestion
    /** @return array<array-key, int> */
    public static function getSubscribedEvents()
```

Reply:
No other service has that return hinting.

Don't forget the well known motto: _To' bien o to' mal, pero to' igua'_

Reply:
Updated with the hint, I have added a task to update getSubscribedEvents in all services :+1: